sirepo/package_data/static/js/controls.js

```diff
@@ -582,16 +582,69 @@ SIREPO.viewLogic('quadrupoleView', function(appState, controlsService, panelStat
     };
 });
 
+SIREPO.app.directive('optimizationPicker', function(latticeService) {
+    return {
+        restrict: 'A',
+        scope: {
+            controller: '='
+        },
+        template: `
+                   <div>
```

Comment:
Last thing - this should be indented only 4 spaces. Everything under it will have to shift over too.